### PR TITLE
Generate custom avatars

### LINKED ISSUE #110: Generate custom avatars
Currently, were using adorable.io API for random avatars. We could change it a bit so we generate octopuses and make them look more in-line with the rest of application

https://github.com/adorableio/avatars-api-middleware

### PR DESCRIPTION
**Pull request by Bombasarkadian**
Closes #110

### CODE REVIEW

File: public/images/octopus-avatar.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<svg id="Octopus" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 146.5 155.3">
```
Comment (Bombasarkadian) [inline public/images/octopus-avatar.svg:1]: This file is a template svg that makes it easier to align other elements

**Comment by Bombasarkadian** [comment]
I'm almost sure it's possible to store all those svg parts in actual svg files that will get bundled into source during `npm build` 🤔

**Comment by Bombasarkadian** [comment]
Or we could actually have a single svg with all of them as hidden layers and simply show the one we actually need

**Comment by pastDexter** [comment]
> I'm almost sure it's possible to store all those svg parts in actual svg files that will get bundled into source during `npm build` 🤔

@Bombasarkadian yes, just put them somewhere in source and use `import avatar from './avatar.svg'`

**Comment by Bombasarkadian** [comment]
@pastDexter It doesn't seem to work this way. I don't get any JS object this way. Instead it "imports" a file path:
```
Array [ "static/media/01.6f66be2d.svg" ]
```

**Comment by Bombasarkadian** [comment]
I got it to work, but it requires a lot of boilerplate code. Do you have any better ideas for this?

File: src/components/user/Avatar/index.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+import bodies from "./parts/bodies";
+import eyes from "./parts/eyes";
+import features from "./parts/features";
+import glasses from "./parts/glasses";
+import borders from "./parts/borders";
+
+const propTypes = {
+  uid: PropTypes.string.isRequired,
+  small: PropTypes.boolean,
```
Comment (Bombasarkadian) [inline src/components/user/Avatar/index.js:12]: That's wrong

File: src/components/user/Avatar/index.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  color: PropTypes.number,
+  bodyStyle: PropTypes.number,
+  body: PropTypes.number,
+  eyes: PropTypes.number,
+  glasses: PropTypes.number,
+  features: PropTypes.number,
+  border: PropTypes.number,
+};
+
+const colors = [
```
Comment (Bombasarkadian) [inline src/components/user/Avatar/index.js:23]: And I don't really like those colors 🙈 We should have something more subtle

**Pull request merged**